src/main/java/io/vlingo/xoom/actors/plugin/mailbox/agronampscarrayqueue/ManyToOneConcurrentArrayQueuePlugin.java

```diff
@@ -26,6 +26,10 @@ public ManyToOneConcurrentArrayQueuePlugin() {
     this.configuration = new ManyToOneConcurrentArrayQueuePluginConfiguration();
   }
 
+  public ManyToOneConcurrentArrayQueuePlugin(PluginConfiguration configuration) {
```

Comment:
Not `final` anymore?

Reply:
@jakzal Oh, I just missed one or some. I am out but can change later, unless you want to.

Reply:
That's what I thought. Just pushed the fix.

Reply:
👍🏻 Thanks!